app/addr_prefixes.go

```diff
@@ -1,13 +1,15 @@
 package app
 
 import (
-	"encoding/binary"
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
 )
 
 const (
+	// AddrLen defines the length of an account's address
+	AddrLen = 20
```

Comment:
Why do we still need 20-byte addresses? Afaict it's only used in one test, is that right? If yes, we can probably fix that test too in this PR.

Reply:
Yeah, this is only used in tests.